modules/check_in/spec/request/v2/sessions_request_spec.rb

```diff
@@ -233,6 +234,65 @@
       end
     end
 
+    context 'when LoROTA returns a 401 for token' do
+      context 'when 401 mapping feature flag is enabled' do
+        let(:resp) do
+          {
+            'errors' => [
+              {
+                'title' => 'Authentication Error',
+                'detail' => 'Authentication Error',
+                'code' => 'LOROTA-MAPPED-API_401',
```

Comment:
The code for `raise_error` middleware uses the key as the code. More specifically, It expects the error response to contain a `code` which will get mapped to the outgoing code, but since LoROTA response doesn't have it, it uses the mapped key as the code: https://github.com/department-of-veterans-affairs/vets-api/blob/039728e708a69c4468fab7796a99c0d3c1fcf396/lib/common/client/middleware/response/raise_error.rb#L37